requirements.txt

```diff
@@ -1,5 +1,4 @@
-wheel==0.23.0
-requests>=2.5.2
-oslo.serialization>=1.4.0
-oslo.log>=1.8.0
+requests==2.5.2
```

Comment:
We should not limit upper border for 'requests',  'oslo.serialization' and 'oslo.log',
because infoblox-client can be used in OpenStack setups and strict version for packages would cause conflicts
with OpenStack requiments:
https://github.com/openstack/requirements/blob/master/global-requirements.txt